src/Sylius/Behat/Page/Shop/HomePage.php

```diff
@@ -89,11 +89,11 @@ public function switchLocale($localeCode): void
         $this->getElement('locale_selector')->clickLink($localeCode);
     }
 
-    public function getLatestProductsNames(): array
+    public function getProductsNames(string $elementName): array
```

Comment:
I'm more into moving this logic into some private method here then using it in respectively dedicated methods like:
```suggestion
    public function getLatestProductsNames(): array
```
and

```suggestion
    public function getLatestDealsNames(): array
```